ide/css.editor/src/org/netbeans/modules/css/editor/ui/CssRuleCreateActionDialog.java

```diff
@@ -135,7 +135,7 @@ private void initComponents() {
         selectElementComboBox = new javax.swing.JComboBox();
         selectClassRadioButton = new javax.swing.JRadioButton();
         selectClassTextField = new javax.swing.JTextField();
-        selectElelmentIdRadioButton = new javax.swing.JRadioButton();
+        selectElementIdRadioButton = new javax.swing.JRadioButton();
```

Comment:
You're changing a variable name, did you change the corresponding .form file as well?

Reply:
you are right here, fixed in new fdad92f1